src/Progress/Progress.js

```diff
@@ -1,22 +1,25 @@
 import React from 'react'
 import _ from 'lodash/fp'
 import { connect } from 'react-redux'
-import { withTranslation } from 'react-i18next'
 import Checklist, { colors } from './Checklist'
 import { selectors, actions } from './Store'
 import needsData from '../Core/needsData'
 import Legend from '../Charting/Legend'
 
 const LegendColor = ({ color }) => <span className="d-inline-block ml-2 mr-1" style={{ backgroundColor: color, width: '1rem', height: '.8rem' }}></span>
 const stateToProps = state => ({ data: selectors.getTree(state) })
-const Progress = _.compose([withTranslation(), needsData(selectors.loaded, actions.load), connect(stateToProps)])(({ t, data }) =>
+
+const Progress = _.compose([needsData(selectors.loaded, actions.load), connect(stateToProps)]) 
```

Comment:
es wird der tag kommen, da werden wir alle dieses blöde compose verstanden haben, und es ganz lcoker flocking in vanilla js umschreiben.... ich warte darauf... 🙈

Reply:
sieht aber ohne translate schon eh viel besser aus hier !